CHANGELOG.md

```diff
@@ -3,6 +3,18 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+## [0.9.5-beta] - 2016-06-16
+### Added
+- Deploy to App Engine flexible compat environment support.
```

Comment:
aha!.

LGTM

On Wed, Jun 15, 2016 at 2:38 PM, Etan Shaul <notifications@github.com>
wrote:

> In CHANGELOG.md
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/735#discussion_r67220446>
> :
>
> > @@ -3,6 +3,18 @@ All notable changes to this project will be documented in this file.
> >
> >  ## [Unreleased]
> >
> > +## [0.9.5-beta] - 2016-06-16
> > +### Added
> > +- Deploy to App Engine flexible compat environment support.
>
> its flex "compat" for this one
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/735/files/1ce83fa8928b40e0406cdf999935a5e1e1fa1521#r67220446>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHf5HYvvs5LvVE76XZ7uu1X_z7LqHgszks5qMEaggaJpZM4I2oK->
> .
>